src/steps/account/account-field-equals.ts

```diff
@@ -13,7 +13,7 @@ export class AccountFieldEquals extends BaseStep implements StepInterface {
 
   protected stepName: string = 'Check a field on a Salesforce Account';
   /* tslint:disable-next-line:max-line-length */
-  protected stepExpression: string = 'the (?<field>[a-zA-Z0-9_]+) field on salesforce account with (?<idField>[a-zA-Z0-9_]+) (?<identifier>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain) (?<expectedValue>.+)';
+  protected stepExpression: string = 'the (?<field>[a-zA-Z0-9_]+) field on salesforce account with (?<idField>[a-zA-Z0-9_]+) (?<identifier>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain|be set|not be set) (?<expectedValue>.+)';
```

Comment:
Couple of things:

- [ ] Note that the order of the operators in this regex expression matters.  Because `be` appears before `be set`, the former will be matched before the latter.  In that case, `operator` is passed to the Cog as `be` and `expectedValue` is passed to the Cog as `set`.
- [ ] Same thing with the `not be set` operator (as it is now, the `operator` would be passed in as `not be` and the `expectedValue` would be `set`
- [ ] In order to make the `expectedValue` optional, you'll also need to add `?` characters after the `expectedValue` capture group, as well as the space before it.

Here's the above demonstrated:

![Screen Shot 2020-05-31 at 6 42 39 PM](https://user-images.githubusercontent.com/3496491/83368734-bd8bb080-a36e-11ea-9d76-fcc4cc0d7df8.png)